tutorials/ph-ug-temp/index.md

```diff
@@ -0,0 +1,106 @@
+
+We can do both steps at the same time by writing a function to do the conversion, keeping the start and end times using the `copyProperties` function, and then by mapping over the `ImageCollection`. Running this code will generate a new `ImageCollection` called 'modLSTc' (MODIS LST converted) with LST in Celsius, along with the specific time periods we need for the chart.
+```
+// convert from Kelvin to Celsius and keep start and end times
+var modLSTc = modLSTday.map(function(img) {
+  return img.multiply(0.02).subtract(273.15).copyProperties(img,['system:time_start','system:time_end']);
+}); // this will apply the correction
+```
+4. Describe land surface temperature data using time series.
+
+It can be helpful to describe your data using a time series graph. We can plot the mean land surface temperature over the year using the following code. Specifically, we create a new chart called 'TS1' (time series chart 1) using the `ui.Chart.image.series` function. This function takes several arguments. First, we bring in our converted `ImageCollection` (modLSTc). Next, we specify the geographic area by calling our 'ug' feature defining the boundary we are interested in. Next, we apply the `ee.Reducer.mean` function to calculate the mean LST for each image. We specify '1000' as a scale (our LST data have 1km resolution). We then specify the 'system:time_start' as the x-axis for our chart.
+
+Calling the `setOptions` function allows us to specify labels for the title and y-axis of the chart.
+```
+// Charts Long-term Time Series
+var ts1 = ui.Chart.image.series(modLSTc, uganda_border,
+ ee.Reducer.mean(), 1000, 'system:time_start').setOptions({
+   title: 'LST Long-Term Time Series',
+   vAxis: {title: 'LST Celsius'},
+ });
+ print(ts1);
+```
+
+5. Visualize processed data on a map.
+
+The last step allowed us to generate a descriptive time series chart. In addition, we may want to visualize our data on the map. We can take the mean LST in celsius, and clip to Uganda. This code will produce a map of the mean temperatures for us to view in the map window.
+```
+var clippedLSTc = modLSTc.mean().clip(uganda_border);
+// Add clipped image layer to map
+Map.addLayer(clippedLSTc, {'min': 0, 'max': 40, 'palette':"blue,limegreen,yellow,darkorange,red"});
+```
+6. Export data for further analysis.
+
+Finally, we need to export our raster image file to perform further analysis (e.g. link to participant or clinic data). We can use the export command below to download the processed image data to our Google Drive folder. Concretely, calling `toDrive` will allow the user to save the exported image in their Google Drive folder. You can specify the location when downloading the image from the "Tasks" window in the console. As we have named the "description" 'LST_Celsius_ug', this will be the name of the exported file. You can change the name by changing the 'description'.
```

Comment:
Please append `()` to function names (e.g., `toDrive()`) here and throughout as a visual cue.

Reply:
Done